packages/app-headless-cms/src/modules/pageTemplates/views/editor/EditorPlugins/SidebarPlugin.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useCallback } from "react";
+import { css } from "emotion";
+import { createComponentPlugin } from "@webiny/app-admin";
+import { Editor } from "~/admin/components/ContentModelEditor";
+import { plugins } from "@webiny/plugins";
+import { CmsEditorFieldTypePlugin } from "~/types";
+import { ReactComponent as ObjectIcon } from "~/admin/icons/ballot_black_24dp.svg";
+import { useContentModelEditor } from "~/admin/components/ContentModelEditor/useContentModelEditor";
+
+const makeScrollable = css`
+    overflow: scroll;
+    height: calc(100vh - 70px);
+`;
+
+export const SidebarPlugin = createComponentPlugin(Editor.Sidebar, OriginalSidebar => {
+    return function Sidebar({ children, ...props }) {
+        const { tabsRef } = useContentModelEditor();
+
+        const onFieldDragStart = useCallback(() => {
+            if (tabsRef.current!.getActiveIndex() > 1) {
```

Comment:
I would rather do a check if tabsRef.current exists than coerce ... If something goes wrong in the parent component and nothing is set into the `.current`, this will break.